api/sendfild/main.go

```diff
@@ -69,6 +69,14 @@ var (
 				Key:      "retry_wait_frequency",
 				DefValue: time.Minute,
 			},
+			"allowedFromAddrs": {
+				Key:      "allowed_from_addrs",
+				DefValue: []string{},
+			},
+			"allowEmptyFromAddrs": {
+				Key:      "allow_empty_from_addrs",
+				DefValue: false,
+			},
```

Comment:
We default to an empty list which means every call to send fil will fail by default. If you want to allow an empty list which is interpreted as "allow all from addresses", you have to explicitly set the `--allowEmptyFromAddrs` flag (which defaults to `false`).

Reply:
How about a single, required, setting. It can be a list of allowed addresses, or special case `*` which allows from any address?

Reply:
>How about a single, required, setting. It can be a list of allowed addresses, or special case * which allows from any address?

Cool, that sounds good to me.

Reply:
Sure that works.